Jenkinsfile.buildchain

```diff
@@ -143,13 +143,13 @@ pipeline {
             script {
                 if(findbugsFile) {
                     echo 'Findbugs reports ...'
-                    findbugs canComputeNew: false, defaultEncoding: '', excludePattern: '', healthy: '', includePattern: '', pattern: findbugsFile, unHealthy: ''
+                    recordIssues enabledForFailure: true, tool: spotBugs(), pattern: findbugsFile
                 }
             }
             script {
                 if(checkstyleFile) {
                     echo 'Checkstyle reports ...'
-                    checkstyle canComputeNew: false, defaultEncoding: '', healthy: '', pattern: checkstyleFile, unHealthy: ''
+                    recordIssues enabledForFailure: true, tool: checkStyle(), pattern: checkstyleFile
```

Comment:
maybe you could extend the command from one line to more to be better readable ;)

Reply:
```suggestion
                    recordIssues(
                      enabledForFailure: true,
                      tool: checkStyle(), 
                      pattern: checkstyleFile
```